lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -5730,6 +5730,9 @@ void SharedEmitterState::gatherFiles(bool separateModules) {
         .Case<om::ClassOp>([&](auto op) {
           symbolCache.addDefinition(op.getSymNameAttr(), op);
         })
+        .Case<om::ConstantOp>([&](auto op) {
```

Comment:
Just wondering, do we create arbitrary OM operations under the top-level operations? I'm not sure it's expected but if so, I think we might want to just accept all OM operations not only limited to constant.